cmd/crio/main.go

```diff
@@ -499,6 +539,58 @@ func main() {
 			Name:  "additional-devices",
 			Usage: fmt.Sprintf("devices to add to the containers (default: %q)", defConf.AdditionalDevices),
 		},
+		cli.StringSliceFlag{
+			Name:  "conmon-env",
+			Usage: fmt.Sprintf("environment variable list for the conmon process, used for passing necessary environment variables to conmon or the runtime (default: %q)", defConf.ConmonEnv),
+		},
+		cli.StringFlag{
+			Name:  "container-attach-socket-dir",
+			Usage: fmt.Sprintf("path to directory for container attach sockets (default: %q)", defConf.ContainerAttachSocketDir),
+		},
+		cli.StringFlag{
+			Name:  "container-exits-dir",
+			Usage: fmt.Sprintf("path to directory in which container exit files are written to by conmon (default: %q)", defConf.ContainerExitsDir),
+		},
+		cli.Int64Flag{
+			Name:  "ctr-stop-timeout",
+			Usage: fmt.Sprintf("the minimal amount of time in seconds to wait before issuing a timeout regarding the proper termination of the container (default: %q)", defConf.CtrStopTimeout),
+		},
+		cli.IntFlag{
+			Name:  "grpc-max-recv-msg-size",
+			Usage: fmt.Sprintf("maximum grpc receive message size in bytes (default: %q)", defConf.GRPCMaxRecvMsgSize),
+		},
+		cli.IntFlag{
+			Name:  "grpc-max-send-msg-size",
+			Usage: fmt.Sprintf("maximum grpc receive message size (default: %q)", defConf.GRPCMaxSendMsgSize),
+		},
+		cli.StringFlag{
+			Name:  "host-ip",
+			Usage: fmt.Sprintf("host IP considered as the primary IP to use by CRI-O for things such as host network IP (default: %q)", defConf.HostIP),
+		},
+		cli.BoolFlag{
+			Name:  "manage-network-ns-lifecycle",
+			Usage: fmt.Sprintf("determines whether we pin and remove network namespace and manage its lifecycle (default: %v)", defConf.ManageNetworkNSLifecycle),
+		},
+		cli.BoolFlag{
+			Name:  "no-pivot",
+			Usage: fmt.Sprintf("if true, the runtime will not use `pivot_root`, but instead use `MS_MOVE` (default: %v)", defConf.NoPivot),
+		},
+		cli.BoolFlag{
+			Name:  "stream-enable-tls",
+			Usage: fmt.Sprintf("enable encrypted TLS transport of the stream server (default: %v)", defConf.StreamEnableTLS),
+		},
+		cli.StringFlag{
+			Name:  "stream-tls-ca",
+			Usage: fmt.Sprintf("path to the x509 CA(s) file used to verify and authenticate client communication with the encrypted stream. This file can change, and CRI-O will automatically pick up the changes within 5 minutes (default: %q)", defConf.StreamTLSCA),
```

Comment:
nit of a nit.  If you've other changes, take it, otherwise good to ignore.
s/The file can change, /The file can change /   (drop comma after change)

Reply:
I'll decided to fix it here since we have a retest nevertheless...